strider/server.py

```diff
@@ -496,6 +501,9 @@ async def lookup(
                 # add new result to hashmap
                 output_results[sub_result_hash] = result
 
+            stop_merging = datetime.datetime.now()
+            message_merging_time += (stop_merging - start_merging).total_seconds()
```

Comment:
I think we should really be using time.time for timing such as these. Not a big deal but FYI